src/Source.zig

```diff
@@ -45,15 +45,15 @@ pub fn lineCol(source: Source, loc: Location) LineCol {
     var start: usize = 0;
     // find the start of the line which is either a newline or a splice
     if (std.mem.lastIndexOfScalar(u8, source.buf[0..loc.byte_offset], '\n')) |some| start = some + 1;
-    const splice_index = for (source.splice_locs, 0..) |splice_offset, i| {
+    const splice_index: u32 = for (source.splice_locs, 0..) |splice_offset, i| {
         if (splice_offset > start) {
             if (splice_offset < loc.byte_offset) {
                 start = splice_offset;
-                break @intCast(u32, i) + 1;
+                break @as(u32, @intCast(i)) + 1;
```

Comment:
Is this `@as` needed?

Reply:
Compilation fails with unknown result type without it